Javascript_2/Candy/script.js

```diff
@@ -0,0 +1,35 @@
+const candyPrices = {
+  sweet: 0.5,
+  chocolate: 0.7,
+  toffee: 1.1,
+  "chewing-gum": 0.03,
+};
+
+const boughtCandyPrices = [];
+
+function addCandy(candyType, weight) {
+  const pricePerGram = candyPrices[candyType];
+  const totalPrice = pricePerGram * weight;
+  boughtCandyPrices.push(totalPrice);
+}
+const amountToSpend = (Math.random() * 100).toFixed();
```

Comment:
Seems like using `toFixed()` makes the variable into a string. To keep it as a number, you can use `Math.floor` instead
```suggestion
const amountToSpend = Math.floor(Math.random() * 100);
```